src/components/community/Base.js

```diff
@@ -13,8 +13,9 @@ const Social = ({ icon, url }) => (
     target="_blank"
     rel="noopener noreferrer"
     className="social-link"
+    aria-label="label"
```

Comment:
This doesn't look right... Did you test this?

Reply:
Tested and updated!